### ISSUE TITLE
Nested View question

### ISSUE THREAD

**Issue by iwarapter** [ghale/gradle-jenkins-plugin#58]
Would it be possible to be able to define a series of nested views in the dsl, for example if we wanted to model:

Nested View
 | -> List View
 | -> Build Pipeline View

**Comment by ghale** [comment]
Yes, it'll support anything the jenkins job dsl will support - see the job dsl reference here: https://github.com/jenkinsci/job-dsl-plugin/wiki/View-Reference#nested-view.
